server/handlers/handlers.go

```diff
@@ -616,6 +625,27 @@ func (ah *ActivityHandler) handleReactions(postID, channelID string, isDirectOrG
 }
 
 func (ah *ActivityHandler) handleDeletedActivity(activityIds clientmodels.ActivityIds) string {
+	if ah.plugin.GetSelectiveSync() {
+		_, chat, err := ah.getMessageAndChatFromActivityIds(&clientmodels.Message{}, activityIds)
```

Comment:
If we need to do this at all, can we just look up the channel information from the post info we already get below? Doesn't seem necessary to make a client call to look it up a different way.